quantum/rgb_matrix_drivers.c

```diff
@@ -41,7 +41,28 @@ static void init(void) {
     IS31FL3731_init(DRIVER_ADDR_4);
 #        endif
 #    elif defined(IS31FL3733)
-    IS31FL3733_init(DRIVER_ADDR_1, 0);
+#        ifndef DRIVER_SYNC_1
+#            define DRIVER_SYNC_1 0
+#        endif
+    IS31FL3733_init(DRIVER_ADDR_1, DRIVER_SYNC_1);
+#        if defined DRIVER_ADDR_2 && (DRIVER_ADDR_1 != DRIVER_ADDR_2)
```

Comment:
Since this PR is targeting develop we might as well update the keyboards too and remove this logic, since even with it, `DRIVER_COUNT` is still set to 2 which probably wastes some RAM:
```
dztech/dz60rgb/v1
dztech/dz60rgb/v2
dztech/dz60rgb_ansi/v1
dztech/dz60rgb_ansi/v2
dztech/dz60rgb_wkl/v1
dztech/dz60rgb_wkl/v1
exclusive/e6_rgb
kbdfans/kbdmini
latin60rgb
miller/gm862
mt64rgb
```